docs/tutorial/t-spark-shell.md

```diff
@@ -0,0 +1,144 @@
+## Interacting with Spark using Interactive Shell
+
+Spark comes with an interactive shell that provides a simple way to learn the API. It is available in either Scala or Python. In this section, we're going to play around a bit with Spark's shell to interact directly with the Spark cluster.
+
+### PySpark Shell
+
+Spark comes with a built-in Python shell where we can execute commands interactively against Spark cluster using Python programming language.
+
+PySpark shell can be launched with `spark-client` snap as:
+
+```bash
+spark-client.pyspark \
+  --username spark --namespace spark
+```
+
+Once the shell is open and ready, you should see a prompt simlar to the following:
+
+```
+Welcome to
+      ____              __
+     / __/__  ___ _____/ /__
+    _\ \/ _ \/ _ `/ __/  '_/
+   /__ / .__/\_,_/_/ /_/\_\   version 3.4.1
+      /_/
+
+Using Python version 3.10.12 (main, Jun 11 2023 05:26:28)
+Spark context Web UI available at http://172.31.29.134:4040
+Spark context available as 'sc' (master = k8s://https://172.31.29.134:16443, app id = spark-b364a0a9a2cb41f3a713c62b900fbd82).
+SparkSession available as 'spark'.
+>>> 
+```
+
+When you open the PySpark shell, Spark spawns a couple of executor pods in the background to process the commands. You can see them by fetching the list of pods in the `spark` namespace.
```

Comment:
```suggestion
When you open the PySpark shell, Spark spawns a couple of executor pods in the background to process the commands. You can see them by fetching the list of pods in the `spark` namespace in a separate shell.
```
Unsure about my wording here, but I basically just wanted to clarify that this is not a command for the pyspark shell that they just opened up.